src/VisualStudio/Xaml/Impl/CodeFixes/AddMissingNamespace/XamlAddMissingNamespacePrefixCodeFixProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Xaml.Features;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.CodeFixes.AddMissingNamespace
+{
+    [ExportCodeFixProvider(StringConstants.XamlLanguageName, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.AddMissingReference)]
+    internal class XamlAddMissingNamespacePrefixCodeFixProvider : CodeFixProvider
```

Comment:
Perhaps Jason thought that we should have added this CodeFixProvider outside of Roslyn.

@jasonmalinowski, our goal is to move more XAML functionality out of our VS assembly and into Roslyn in order to provide more functionality and share more code.